### PR TITLE
:memo: reactions

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

File: packages/k-ramel/doc/REACTIONS.md [hunk 1]
```diff
@@ -16,3 +16,66 @@
     <a href="./STORE.md">store</a>
   &nbsp;]
 </p>
+
+## What is a reaction?
+A reaction is something your plug to a listeners (see [listeners' documentation here](./LISTENERS.md)).\
```
Comment (bpetetot) [inline packages/k-ramel/doc/REACTIONS.md]: *A reaction is a function plugged to a listener*

File: packages/k-ramel/doc/REACTIONS.md [hunk 2]
```diff
@@ -16,3 +16,66 @@
...
+This is where you write your logical code.\
+In a reaction you can access:
+ - the action that trigger your reaction (the action that your listener has filtered)
```
Comment (bpetetot) [inline packages/k-ramel/doc/REACTIONS.md]: _(the action that your listener has caught)_

File: packages/k-ramel/doc/REACTIONS.md [hunk 3]
```diff
@@ -16,3 +16,66 @@
...
+## Example & API
+```js
+import { when } from 'k-ramel'
+
+// a reacton is a function
+const reaction = () => {
+  console.log('I am a reaction!')
+}
+
+// you can acces the action that triggered your reaction
```
Comment (bpetetot) [inline packages/k-ramel/doc/REACTIONS.md]: // you can **access** the action that **has** triggered your reaction

File: packages/k-ramel/doc/REACTIONS.md [hunk 4]
```diff
@@ -16,3 +16,66 @@
...
+ - the store (the k-ramel store, with dispatch, data, drivers, etc)
+ - the drivers
+
+A reaction is a function whom the signature is: `(action, store, drivers)`.
+
+## Example & API
+```js
+import { when } from 'k-ramel'
+
+// a reacton is a function
```
Comment (bpetetot) [inline packages/k-ramel/doc/REACTIONS.md]: // a **reaction** is a function

File: packages/k-ramel/doc/REACTIONS.md [hunk 5]
```diff
@@ -16,3 +16,66 @@
...
+
+// you can acces the action that triggered your reaction
+const reaction = (action) => {
+  const { type, payload } = action
+
+  console.log('I am triggered by the action:', type)
+  console.log('\tthe payload is:', payload)
+}
+
+// you can also acces the store, to read the state, or modify it, or dispatch a new event, etc
```
Comment (bpetetot) [inline packages/k-ramel/doc/REACTIONS.md]: // you can also **access** the store, to read the state, or modify it, or dispatch a new event, etc

File: packages/k-ramel/doc/REACTIONS.md [hunk 6]
```diff
@@ -16,3 +16,66 @@
...
+const reaction = (action, store) => {
+  const users = action.payload
+  const usersSortedByName = users.sort((first, second) => first.name < second.name)
+
+  store.ui.users.sortedByName.set(usersSortedByName)
+
+  store.dispatch('USERS_SORTED_BY_NAME')
+}
+
+// you can acces the drivers!
```
Comment (bpetetot) [inline packages/k-ramel/doc/REACTIONS.md]: // you can **access** the drivers!

**Comment by fabienjuif** [comment]
Thank you @bpetetot 
I am starting to be tired in this documentation 😴

**Comment by fabienjuif** [comment]
Thank your @bpetetot 💋

**Pull request merged**